content/src/main/content/jcr_root/apps/acs-commons/authoring/sitewide-component-props/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+      jcr:primaryType="cq:ClientLibraryFolder"
+      categories="[cq.authoring.editor]"
+      dependencies="[granite.shared,granite.ui.foundation.admin,rte.coralui2,coralui2.imageeditor,history,lawnchair,granite.ui.notifications,granite.ui.annotations,cq.siteadmin.common.notifications,cq.social.commons]"/>
```

Comment:
This is now refactored on the HS2 repo. Once we finalize our updates I'll update this PR.